### ISSUE TITLE
changing display settings doesn't save them

### ISSUE THREAD

**Issue by nodaba** [WayfireWM/wayfire#675]
Congratulations, I'm running a new version of wayfire now.
I installed a tool that changes the display settings on sway and does so with wayfire too,
it is called wdisplays. I successfully changes the display settings but doesn't save them.

**Comment by ammen99** [comment]
There are two ways to configure your outputs and make that permanent:

1. Use wayfire's config file: https://github.com/WayfireWM/wayfire/wiki/Configuration#output
Note you will need to use this if you want to enable screen mirroring.

2. Use [kanshi](https://github.com/emersion/kanshi). If you want to integrate that with `wdisplays`, then try https://github.com/cyclopsian/wdisplays/pull/5

Everything else that you do is temporary like with `xrandr`

**Comment by nodaba** [comment]
oh, wouldn't it be possible that wayfire saves the new resolution when I change it?

**Comment by ammen99** [comment]
This is possible, but I think you have the wrong impression. Changing resolution with `wdisplays` is like changing resolution with `xrandr` on X11 - these changes are not permanent.

Setting resolution with `kanshi` or wayfire's config file is like changing resolution in `gnome-control-center`, i.e it is permanent.

There are use-cases for both permanent and temporary resolution changes. This is why we won't save the resolution set by `wdisplays` except if it doesn't save it as a kanshi profile.

TLDR: use kanshi or config file
